cmd/mountlib/mount.go

```diff
@@ -356,6 +356,12 @@ infrastructure](https://github.com/billziss-gh/winfsp/wiki/WinFsp-Service-Archit
 which creates drives accessible for everyone on the system or
 alternatively using [the nssm service manager](https://nssm.cc/usage).
 
+Note also that filesystem paths in mounts are not automatically converted
```

Comment:
I'm not quite sure what I meant by saying that filesystem paths in mount are not automatically converted...

We do since 28255f1bacb2b4c46a0fd002b7477a92e0144c5b use ~~UNC~~ extended path format files in the VFS cache.

As to whether the external interface to mount as in when you are using `Z:` on Windows works with extended path format - I expect it does but I don't know whether WInFSP supports it.

Can you check? If you mount a google drive - can you make file names with longer than 259 characters and access then from Windows. I suspect the answer will be Yes.

So I think we should proabably drop this paragraph until we know what we are talking about, as I certainly don't!

Reply:
I did some quick testing, and seems mount handles long paths without any apparant limitations. Including writing to paths longer than 259 characters within a mounted drive letter, and even even crazy things like  `rclone.exe mount [C:\total path of 299 characters] X:`. Has not done a very thorough test, so there could still be issues, but I don't think I will invest the time cover everything...